src/app/InteractionModelEngine.h

```diff
@@ -451,6 +462,29 @@ class InteractionModelEngine : public Messaging::UnsolicitedMessageHandler,
     bool EnsureResourceForSubscription(FabricIndex aFabricIndex, size_t aRequestedAttributePathCount,
                                        size_t aRequestedEventPathCount);
 
+    /**
+     * Verify and ensure (by killing oldest read handlers that make the resources used by the current fabric exceed the fabric
+     * quota) the resources for handling a new read transaction with the given resource requirments.
+     * - PASE sessions will be counted in a virtual fabric (i.e. kInvalidFabricIndex will be consided as a "valid" fabric in this
+     * function)
+     * - If the existing resources can serve this read transaction, this function will return Status::Success.
+     * - or if the resources used by read transactions in the fabric index meets the per fabric resource limit (i.e. 9 paths & 1
+     * read) after accepting this read request, this function will always return Status::Success by evicting existing read
+     * transactions which uses more resources than kMinSupportedPathsPerReadRequest from the oldest to the newest.
```

Comment:
```suggestion
     * transactions from other fabrics which are using more than the spec-guaranteed minimum number of read
     * requests or paths. Read requests will be prioritized for eviction based on whether they use more
     * paths than the spec-mandated limits and then based on their age, with newest reads being
     * evicted first.
```
or so?

Reply:
"spec-guaranteed minimum number" -> "guaranteed minimal number"

There is a `GetGuaranteedReadRequestsPerFabric` function below for the value, we are actually guaranteed resources no less than the requirement from the spec.